tools/base/fpgainfo/board.h

```diff
@@ -0,0 +1,73 @@
+// Copyright(c) 2019, Intel Corporation

```

Comment:
General comment on board.h/board.c files.
Are these meant for containing generic plugin-based fpgainfo framework? If so, are mac/phy applicable to generic OPAE compatible FPGAs?

Reply:
generic plugin-based fpgainfo framework + Board / Platform specific information